sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -147,6 +147,9 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
   test("SPARK-14415: All functions should have own descriptions") {
     for (f <- spark.sessionState.functionRegistry.listFunction()) {
       if (!Seq("cube", "grouping", "grouping_id", "rollup").contains(f.unquotedString)) {
+        if (f.unquotedString == "bloom_filter_agg") {
```

Comment:
do we need this? it looks weird to handle `bloom_filter_agg` here.

Reply:
I was debugging with checkpointing here with IDE 😂